src/main/java/kr/codesqaud/cafe/board/controller/CommentController.java

```diff
@@ -16,15 +18,18 @@ public CommentController(CommentService commentService) {
     }
 
     @PostMapping("/board/{postId}/comment")
```

Comment:
RestFul URL를 한번 고민하는 것을 추천드립니다. Post 가 이미 해당 컬렉션에 속하는 새로운 자원을 생성한다. 자원의 URI는 시스템에 의해 할당하다는 의미가 있어서 이렇게 하면 어떻까요?
- 참조
https://spoqa.github.io/2012/02/27/rest-introduction.html

```suggestion
    @PostMapping("/boards/{postId}")
```